CONTRIBUTORS.md

```diff
@@ -5,32 +5,45 @@
 | @abmsourav | @sourav926 |
 | @aristath | @aristath |
 | @audrasjb | @audrasjb |
+| @beafialho | @beafialho |
 | @bgardner | @bgardner |
 | @briceduclos | @briceduclos |
 | @carolinan | @poena |
+| @clucasrowlands | @clucasrowlands |
 | @colorful-tones | @colorful-tones |
+| @desrosj | @desrosj |
 | @dgwyer | @dgwyer |
 | @dimadin | @dimadin |
 | @gregrickaby | @gregrickaby |
 | @ellenbauer | @ellenbauer |
+| @felixarntz | @flixos90 |
+| @jasmussen | @joen |
 | @jffng | @jffng |
-| @iwangdu | @ |
+| @juricav | UNKNOWN |
+| @iwangdu | UNKNOWN |
 | @kafleg | @kafleg |
 | @karmatosed | @karmatosed |
 | @kraftbj | @kraftbj |
 | @kjellr | @kjellr |
 | @littlebigthing | @littlebigthing |
+| @luminuu | @luminuu |
 | @maggiecabrera | @onemaggie |
+| @melchoyce | @melchoyce |
 | @mtias | @matveb |
 | @mburridge | @mburridge |
 | @mtoensing | @mtoensing |
 | @nickcernis | @nickcernis |
 | @nielslange | @nielslange |
+| @ntwb | @netweb |
+| @otto42 | @otto42 |
 | @pattonwebz | @williampatton |
 | @pbking | @pbking |
 | @ribaricplusplus | @ribaricplusplus |
 | @richtabor | @onemaggie |
```

Comment:
would you mind changing this to use Rich's wp.org username? looks like a copy-paste error here :D